Tools/Scripts/webkitpy/benchmark_runner/webserver_benchmark_runner.py

```diff
@@ -57,15 +57,15 @@ def _construct_subtest_url(self, subtests):
         return subtest_url
 
     def _run_one_test(self, web_root, test_file, iteration):
-        enable_profiling = True if self._profile_output_dir else False
+        enable_profiling = True if self._profile_output_dir and self._trace_type else False
```

Comment:
Maybe we can just do
`enable_profiling = self._profile_output_dir and self._trace_type `